### ISSUE TITLE
Move to Qt 6

### ISSUE THREAD

**Issue by aaronhktan** [aaronhktan/jyut-dict#136]
Qt 5 is super outdated by now. This project should move to Qt 6 (there's already a branch that does that but it's fallen into disrepair).

This entails:
- Moving from qmake to CMake since qmake is probably not long for this world
- Dropping support for Windows 7 and 8
- Deleting the various Qt5 checks in the codebase

**Comment by vintotal** [comment]
Is it possible that a new requirment for the windows version is also Direct X 12?

**Comment by aaronhktan** [comment]
> Est-il possible que la dernière version (bêta) demande également DirectX 12 sur Windows?

Non, je ne crois pas. Cette version a été compilée avec Qt 5.15, soit la même version de Qt que la version précédente.

**Comment by aaronhktan** [comment]
Update: compilation works on Qt 6, speech to text appears to be broken on macOS. (See https://github.com/aaronhktan/jyut-dict/tree/feature/aaronhktan/qt-6) Will not be moving to Qt 6 until this is fixed.
